css/style.css

```diff
@@ -0,0 +1,66 @@
+body {
+    background-color: #39393A;
+    color: #C5DECD;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 28px;
+}
+
+h1, h2 {
+    font-family: 'Jost', sans-serif;
+    font-size: 38px;
+}
+
+p {
+    color: #FDCA40;
+}
+
+li {
+    list-style-type: square;
+    color: #AEDD92;
+}
+
+img {
+    margin-left: 5px;
+}
+
+input {
+    padding: 10px;
+    border-radius: 5px;
+}
+
+input[type=text] {
+    width: 250px;
+    border: none;
+}
+
+input[type=submit], button {
+    color: #39393A;
+    border: none;
+    font-size: 18px;
+}
+
+input[type=submit] {
+    background-color: #C5DECD;
+}
+
+button {
+    background-color: #FDCA40;
+}
+
+input[type=submit]:hover, button:hover{
+    background-color: #FFFFFF;
+    cursor: pointer;
+}
+
+footer {
+    margin-top: 10px;
+    font-size: 18px;
+}
+
+a {
+    color: #70D7FF;
+}
+
+a:hover {
+    color: #FFFFFF;
+}
```

Comment:
I like that this CSS file is organized with the selectors generally in the order they appear on the page - it makes it easier to find where the rule for a given element is within the file.

Reply:
Thanks! I'm trying to find better ways to organize my css selectors.